t/traffic.t

```diff
@@ -130,3 +143,69 @@ $/s
 --- no_error_log
 [error]
 [lua]
+
+
+
+=== TEST 3: block by limit-count (output states)
+--- http_config eval
+"
+$::HttpConfig
+
+    lua_shared_dict req 1m;
+    lua_shared_dict conn 1m;
+    lua_shared_dict count 1m;
+"
+--- config
+    location = /t {
+        content_by_lua '
+            local limit_conn = require "resty.limit.conn"
+            local limit_req = require "resty.limit.req"
+            local limit_count = require "resty.limit.count"
+            local limit_traffic = require "resty.limit.traffic"
+
+            local lim1 = limit_req.new("req", 3, 2)
+            local lim2 = limit_req.new("req", 2, 3)
+            local lim3 = limit_conn.new("conn", 4, 1, 2)
+            local lim4 = limit_count.new("count", 2, 100)
+
+            local limiters = {lim1, lim2, lim3, lim4}
+
+            ngx.shared.req:flush_all()
+            ngx.shared.conn:flush_all()
+            ngx.shared.count:flush_all()
+
+            local keys = {"foo", "bar", "foo", "bar"}
+            local states = {}
+            for i = 1, 6 do
+                local delay, err = limit_traffic.combine(limiters, keys, states)
+                if not delay then
+                    ngx.say("failed to limit traffic: ", err)
+                    ngx.say("states: ", table.concat(states, ", "))
+                else
+                    ngx.say(i, ": ", delay,
+                            ", conn committed: ", lim3:is_committed(),
+                            ", states: ", table.concat(states, ", "))
+                end
+                if i == 4 then
+                    ngx.sleep(1)
```

Comment:
This extraneous delay doesn't seem necessary to make this test succeed, is it? Considering we are working in a 100s window.

Reply:
@thibaultcha thanks for your review, fixed all of them.